### PR TITLE
Add basic thread-safe time measurement to the logger

### PR DESCRIPTION
**Pull request by PawelLipski**
image

### CODE REVIEW

File: logging/src/main/java/com/virtuslab/logger/PrefixedLambdaLogger.java [hunk 1]
```diff
@@ -5,10 +5,13 @@
 import java.util.function.Supplier;
 
 import kr.pe.kwonnam.slf4jlambda.LambdaLogger;
+import org.checkerframework.checker.nullness.qual.Nullable;
 
 public class PrefixedLambdaLogger implements IPrefixedLambdaLogger {
   private final LambdaLogger logger;
   private final String className;
+  private final ThreadLocal<@Nullable String> bufferedTimerMessage;
```
Comment (PawelLipski) [inline logging/src/main/java/com/virtuslab/logger/PrefixedLambdaLogger.java:13]: Tip: `ThreadLocal<T>` is easiest to conceptualize as `ConcurrentHashMap<Long /* thread id */, T>`

File: logging/src/main/java/com/virtuslab/logger/PrefixedLambdaLogger.java [hunk 2]
```diff
@@ -34,7 +39,9 @@ private String getLogMessagePrefix() {
     // #6: method from which IMacheteLogger.<logMethod> was called;
     @SuppressWarnings("upperbound")
     StackTraceElement element = Thread.currentThread().getStackTrace()[6];
-    return className + "#" + element.getMethodName() + ": ";
+    String timerMessage = bufferedTimerMessage.get();
```
Comment (PawelLipski) [inline logging/src/main/java/com/virtuslab/logger/PrefixedLambdaLogger.java:42]: ... and then `ThreadLocal#get` would be just sth like `.get(Thread.currentThread().getId())`

Comment (PawelLipski) [reply logging/src/main/java/com/virtuslab/logger/PrefixedLambdaLogger.java:42]: Similarly with `set` and `remove`

File: scripts/utils.sh [hunk 3]
```diff
@@ -16,7 +16,15 @@ function die() {
 function extract_version_from_gradle_file_stdin() {
   [[ -t 0 ]] && die "${FUNCNAME[0]}: expecting non-terminal stdin, aborting" || true
 
-  grep -Po "(?<=PLUGIN_VERSION = ').*(?=')"
+  # Let's avoid any external process calls (cat/grep) to speed things up.
```
Comment (mkondratek) [inline scripts/utils.sh:19]: :heart_decoration:

**Pull request merged**